src/ltablib.js

```diff
@@ -291,7 +291,7 @@ const auxsort = function(L, lo, up, rnd) {
 const sort = function(L) {
     let n = aux_getn(L, 1, TAB_RW);
     if (n > 1) {  /* non-trivial interval? */
-        lauxlib.luaL_argcheck(L, n < llimit.MAX_INT, 1, lua.to_luastring("array too big", true));
+        lauxlib.luaL_argcheck(L, n < Number.MAX_SAFE_INTEGER, 1, lua.to_luastring("array too big", true));
```

Comment:
I guess this is the change breaking the test?
The test in sort.js compares this to math.maxinteger.

I think the test is wrong. I'll ask roberto about it.

Reply:
I forgot to ask roberto at the workshop. emailed lua list instead.